exercise_3.java

```diff
@@ -0,0 +1,130 @@
+import java.util.Random;
+import java.util.Scanner;
+
+public class exercise_3 {

```

Comment:
```suggestion
public class Exercise3 {

```
в java всё, кроме static final полей именуются по принципу camelCase